verifier/src/quote.rs

```diff
@@ -28,9 +28,10 @@ impl<T> Quote3Verifier<T> {
 impl<T: AsRef<[u8]>, E: Accessor<Quote3<T>>> Verifier<E> for Quote3Verifier<T> {
     type Value = ();
     fn verify(&self, evidence: &E) -> VerificationOutput<Self::Value> {
-        let quote = evidence.get();
-        let is_success = quote.verify(&self.key).is_ok() as u8;
-
+        let is_success = self
+            .key
+            .map(|key| evidence.get().verify(&key).is_ok() as u8)
+            .unwrap_or(0);
```

Comment:
I'm ok if people feel this is too dense and should be broken out more.

Reply:
I'm fine with it, its pretty easy to follow and we've done similar stuff all over.